entries/h/hackster.io.json

```
@@ -0,0 +1,11 @@
+{
+  "Hackster.io": {
+    "domain": "hackster.io",
+    "contact": {
+      "twitter": "Hacksterio"
+    },
+    "keywords": [
+        "education"
```

I don't think the education category is a very good fit but neither is the creativity or social category. Maybe it's better to put it in the _other_ category until a more fitting one is created.
```suggestion
        "other"
```